### DISCUSSION TITLE
An error occured

### DISCUSSION THREAD

**Discussion by IguodalaYan** [LooseLab/readfish#256] [Q&A] [answered]
![b870a79fb9e6a6fe6ed665cdf2b7bb5](https://github.com/LooseLab/readfish/assets/143084410/ae30cd46-1815-452b-b591-eae292bbbd7f)
Please help!

**Comment by mattloose** [comment, marked as answer]
Hi,

The error message is telling you that the position has no run started on it. You have to start the run first and then start readfish.

Hope that helps.

**Reply by IguodalaYan** [reply]
Thanks!
